src/commands/force/source/delete.ts

```diff
@@ -146,14 +147,12 @@ export class Delete extends DeployCommand {
       // determine if user is trying to delete a single file from a bundle, which is actually just an fs delete operation
       // and then a constructive deploy on the "new" bundle
       this.components
-        .filter((comp) => {
-          if (comp.type.strategies?.adapter === 'bundle' && comp instanceof SourceComponent) return true;
-        })
+        .filter((comp) => comp.type.strategies?.adapter === 'bundle' && comp instanceof SourceComponent)
         .map((bundle: SourceComponent) => {
-          sourcepaths.map((sourcepath) => {
+          sourcepaths.map(async (sourcepath) => {
```

Comment:
does that actually work?  Seems like there should be a Promise.all outside of it

Reply:
I haven't seen any issues from this... but I agree 🤔